docs/content/components/footer.mdx

```diff
@@ -85,7 +85,7 @@ code_snippets:
           class="usa-footer usa-footer--nci-big"
           id="nci-footer"
           aria-label="Footer">
-          <div class="grid-container usa-footer__return-to-top">
+          <div class="usa-footer__nci-return-to-top">
```

Comment:
The html for back to top footer includes this grid container class. Is this causing problems for you? If it's causing problems, we would need to remove it for everything. Otherwise, let's keep the html the same.

Reply:
never mind,

grid container is needed for the default back to stop state before the hovering button is initialized. this does not apply.